e2e/run.go

```diff
@@ -25,7 +26,7 @@ func GoRun(ctx context.Context, out, errOut io.Writer, args ...string) (int, err
 		return 0, err
 	}
 	go func() {
-		cleanupOnDone(ctx.Done(), cmd.Process.Pid, errOut)
+		cleanupOnDone(ctx.Done(), cmd.Process.Pid, os.Stderr)
```

Comment:
Doesn't look like there are any other callers of `cleanupOnDone`. Could remove the param altogether and just use Stderr in the function itself

Reply:
good catch, updated